public/app/percona/add-instance/components/AddRemoteInstance/FormParts/FormParts.messages.ts

```diff
@@ -26,6 +26,14 @@ export const Messages = {
         username: 'Username',
         password: 'Password',
       },
+      postgresqlDetails: {
+        address: 'Hostname',
```

Comment:
Only leave `database` in `postgresqlDetails` and use the others from `mainDetails` to avoid duplication. (for the other down as well)